app/views/hiring_staff/vacancies/review.html.haml

```diff
@@ -1,5 +1,4 @@
-- content_for :page_title_prefix, "Review the job details — Publish a job for #{@school.name}"
-
+- content_for :page_title_prefix, "#{@vacancy.review_page_title}"
```

Comment:
I think you shouldnt need to wrap the vacancy method in quotes and evaluate. Other than that looks great!

Reply:
Ah yeah, good catch, thanks! 👍